#length(data) <= 100
uint8[50] data
---
bool result
